eks-distro-base/check_update.sh

```diff
@@ -18,12 +18,14 @@ set -e
 set -o pipefail
 set -x
 
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+
 IMAGE_REPO=$1
 IMAGE_NAME=$2
 IMAGE_TAG=$3
 DRY_RUN_FLAG=$4
 
-BASE_IMAGE=${IMAGE_REPO}/${IMAGE_NAME}:$(cat TAG_FILE)
+BASE_IMAGE=${IMAGE_REPO}/${IMAGE_NAME}:$(cat ${REPO_ROOT}/../EKS_DISTRO_BASE_TAG_FILE)
```

Comment:
Why are we going back a level from the REPO_ROOT? I thought REPO_ROOT is the furthest we can go right?

Reply:
REPO_ROOT actually points to the directory in which the script which is being invoked is located. Maybe change the name to SCRIPT_DIR?

Reply:
Yup let's call it SCRIPT_DIR or BASE_DIR depending on what makes sense. In that case, just double check whether the naming is consistent with the other files as well